mod/01_core/spec/set/all/templating_spec.rb

```diff
@@ -69,7 +70,9 @@
     end
     
     it "should apply to new cards" do
-      expect(Card.new(:name=>"Pete+birthday").content).to eq('Today!')
+      pb = Card.new :name=>"Pete+birthday"
+      expect(pb.raw_content).to eq('Today!')
```

Comment:
The real deeper issue is developer documentation, where it is and should be, and creating tickets to work on it at the right moment in time.  I started to write a book http://gerry.wagn.org/It_s_Cards_all_the_Way_Down where I'm starting from a more general perspective, but ultimately we should be able to attach more detail to each part.  I intend to use our technology to make that detail *multi-dimensional*, each component of **REWD** being one of the dimensions (at a minimum).

It is a collective work in progress, so come on in and edit, add content or discussions to any of it.